config.json

```diff
@@ -590,4 +593,4 @@
       }
     }
   ]
-}
+}
```

Comment:
it's been added.  see https://github.com/mjschock/stencil/blob/f80cdfc06862a9668db449697261c8d9a07eb4a2/config.json.  
![screen shot 2017-01-25 at 10 19 04 am](https://cloud.githubusercontent.com/assets/1357197/22303252/488ed85c-e2e7-11e6-8a2d-ec08af9cd80f.png)